schema/partners.js

```diff
@@ -36,8 +37,7 @@ const Partners = {
       `,
     },
     type: {
-      type: new GraphQLList(GraphQLString),
-      description: 'Only return partners of the specified _type(s).',
+      type: new GraphQLList(PartnerClassification.type),
```

Comment:
I'm personally not convinced this is less confusing since you're still naming the arg `type`.

You *could* remap the entire thing to `kind` and call them `PartnerKinds`but I also think just calling them Types and going with it is fine.